modules/cluster-logging-configuration-of-json-log-data-for-default-elasticsearch.adoc

```diff
@@ -79,24 +79,30 @@ Suppose that you use the following snippet in your `ClusterLogForwarder` CR YAML
 
 [source,yaml]
 ----
-outputDefaults:
+outputDefaults: <1>
  elasticsearch:
-    structuredTypeKey: openshift.labels.myLabel <1>
+    structuredTypeKey: kubernetes.labels.managed <2>
     structuredTypeName: nologformat
+outputs:
+  - name: elasticsearch-secure
```

Comment:
structured Field are defined for default elasicsearch..

Reply:
For  example. 
send json  pod logs  under project-qa-x to index:  app-qa-000000x
send json  pod logs  under project-dev-x to index:  app-dev-000000x
```
apiVersion: logging.openshift.io/v1
kind: ClusterLogForwarder
metadata:
  name: instance
  namespace: openshift-logging
spec:
  inputs: 
    - name: QELogData
      application:
        namespaces:
        - project-qa1
        - project-qa2
          .....
    - name: DevLogData
      application:
        namespaces:
        - project-dev1
        - project-dev2
          ....
  outputDefaults:
    elasticsearch:
      structuredTypeKey: openshift.labels.mylabel
      structuredTypeName: nologformat
  pipelines:
    - name: pipe_first
      inputRefs:
        - QELogData
      outputRefs:
        - default
      labels:
        mylabel: qa
      parse: json
    - name: pipe_second
      inputRefs:
        - DevLogData
      outputRefs:
        - default
      parse: json
      labels:
        mylabel: dev
```

Reply:
Made updates as discussed in slack thread: https://redhat-internal.slack.com/archives/GGUR75P60/p1718034341293289